File: java/1008.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in); 
        int num = in.nextInt();
        int hour = in.nextInt();
        double salary_per_hour = in.nextDouble();

        System.out.printf("NUMBER = %d\n", num);
        System.out.printf("SALARY = U$ %.2f\n", hour * salary_per_hour);
    }
}

File: kotlin/1008.kt
import java.util.*

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`);
    
    val num: Int = sc.next().toInt();
    val hour: Int = sc.next().toInt();
    val salary_per_hour: Double = sc.next().toDouble();
    
    println("NUMBER = ${num}");
    println("SALARY = U$ ${(hour * salary_per_hour).format(2)}");
}

fun Double.format(digits: Int) = "%.${digits}f".format(this)